core/equations.gms

```diff
@@ -591,6 +591,12 @@ q_emiTeMkt(t,regi,emiTe(enty),emiMkt)..
 		  vm_emiIndCCS(t,regi,emiInd37_fuel)
 		)$( sameas(enty,"co2") )
 	)$(sameas(emiMkt,"ETS"))
+*** substract carbon in feedstocks from biogenic or synthetic origin, generates negative emissions
```

Comment:
Hey! I suggest that we stick to the `C2CCUshort` accounting idea and we create a `C2CCUlong` variable. `FeedstocksCarbon` would be part of that variable but not the only component (later). I'll propose an implementation.

Reply:
Also, I think with the definition of `vm_FeedstocksCarbon`, we're assuming that all the carbon content of the industry's products is never going back into the atmosphere. Unless we account for those emissions somewhere else (decomposition of urea, waste incineration, waste power, ...). I'm on it!

Reply:
> Hey! I suggest that we stick to the C2CCUshort accounting idea and we create a C2CCUlong variable. FeedstocksCarbon would be part of that variable but not the only component (later). I'll propose an implementation.

Ok, I see. But, I guess you could to is based on this implementation, or not? So that vm_FeedStocksCarbon or a certain share of it becomes part of this long-term CCU variable?

> Also, I think with the definition of vm_FeedstocksCarbon, we're assuming that all the carbon content of the industry's products is never going back into the atmosphere. Unless we account for those emissions somewhere else (decomposition of urea, waste incineration, waste power, ...). I'm on it!

Right! However, I think, we should be pragmatic for a first solution here because we need non-energy use in the model soon. A (fix) share of feedstocks carbon going to long-term storage might even do. 

I'd say that what we need in the end as a first step is that we have a roughly matching historic FE non-energy use in Germany, industry emissions corrected for feedstocks carbon and a basic dynamic to reduce fossil feedstocks in a policy scenario.

Reply:
Yes! I would definitely implement CCUlong based on vm_FeeedstocksCarbon. I left a proposal for that in the Mattermost channel but I'm going to implement it today to try it out and I'll share it :)